be/src/util/tdigest.h

```diff
@@ -72,27 +72,13 @@ const size_t kHighWater = 40000;
 class Centroid {
```

Comment:
1. this definition should be moved to source file too.
2. including should be moved to source file too to make header file inlcude files as few as possible.